packages/core/src/RenderingEngine/VolumeViewport.ts

```diff
@@ -242,19 +242,20 @@ class VolumeViewport extends BaseVolumeViewport {
   /**
    * Reset the camera for the volume viewport
    */
-  public resetCamera(
-    resetPan = true,
-    resetZoom = true,
-    resetToCenter = true,
-    resetRotation = false,
-    supressEvents = false,
-    resetOrientation = true
-  ): boolean {
+  public resetCamera(options?): boolean {
+    const {
+      resetPan = true,
+      resetZoom = true,
+      resetRotation = true,
+      resetToCenter = true,
+      suppressEvents = false,
+      resetOrientation = true,
+    } = options || {};
     const { orientation } = this.viewportProperties;
     if (orientation && resetOrientation) {
       this.applyViewOrientation(orientation, false);
     }
-    super.resetCamera(resetPan, resetZoom, resetToCenter);
+    super.resetCamera({ resetPan, resetZoom, resetToCenter });
```

Comment:
Can you call the parent method with the same options as are incoming, and assign a fixed value to options - that way the super method can add new options in the future and it can still work as expected.